tv_apps/notification-receiver/js/app.js

```diff
@@ -0,0 +1,15 @@
+/* global Receiver */
+'use strict';
+
+(function(exports) {
+
+  exports.receiver = new Receiver();
+
+  var onLoad = function() {
```

Comment:
Based on previous offline discussion with Gecko developer, page load and presentation session creation are two separated tasks which are processed in parallel. For gaia I think it is better to have everything ready before touching `navigator.presentation.session`.
This basically follows the same design as in fling-player at https://github.com/mozilla-b2g/gaia/pull/27624/files#diff-6fc639e67a9b67d25ad71ff8799ce847R59